virttest/qemu_vm.py

```diff
@@ -1152,6 +1152,14 @@ def add_cpu_flags(devices, cpu_model, flags=None, vendor_id=None,
                     if not flags.startswith(","):
                         cmd += ","
                     cmd += "%s" % flags
+                    # Win10 and Win216 guest required 'erms' flag,
+                    # if cpu model is Penryn or Nehalem. see detail
+                    # in RHBZ#1252134
+                    if cpu_model in ['Penryn', 'Nehalem']:
+                        recognize_flags = utils_misc.get_recognized_cpuid_flags(
+                            qemu_binary)
+                        if not ('erms' in flags or 'erms' in recognize_flags):
```

Comment:
I think erms only can be exported when it's supported in host and qemu
so should be 
'erms' in flags and 'erms' in recognize_flags